website/content/ja/wgs/platforms/maturity-model/v1/index.md

```diff
@@ -15,7 +15,7 @@ window.onhashchange = function() {
   // get the fragment without the `#`
   const fragment = window.location.hash.substring(1)
   const found = Array.from(document.querySelectorAll('.nav-item'))
-    .filter(el => el.textContent === fragment)
+    .filter(el => el.textContent === decodeURI(fragment))
```

Comment:
This change makes the tab switcher of the model table work even if the tab names are written in multi-byte characters.